pages/public_cloud/containers_orchestration/managed_kubernetes/cluster-autoscaler-example/guide.en-gb.md

```diff
@@ -29,305 +29,240 @@ You can activate the autoscaler on several node pools, each of which can have a
-status:
-  availableNodes: 3
-  conditions:
-    [...]
-  currentNodes: 3
-  observedGeneration: 2
-  upToDateNodes: 3
-```
-
-In the `spec` section you can see that the `autoscale` parameter is set to `false`. In order to enable the autoscaler, you need to patch the node pool to set this field to `true`. Set also `desiredNodes` to 1, `minNodes` to 1 and `maxNodes` to 3, to be sure that we begin only with one active node and that we can grow the node pool up to 3 nodes.
-
-```bash
-kubectl patch nodepool <your_nodepool_name> --type="merge" --patch='{"spec": {"autoscale": true, "desiredNodes": 1, "minNodes": 0, "maxNodes": 3}}'
-```
-
-As you can see in my example, patching the node pool definition enables the autoscaler:
-
-```console
-$ kubectl patch nodepool nodepool-b2-7 --type="merge" --patch='{"spec": {"autoscale": true}}'
-nodepool.kube.cloud.ovh.com/nodepool-b2-7 patched
+> [!primary]
+>
+> The source code of the following example is available in the public Github repository [public-cloud-examples](https://github.com/ovh/public-cloud-examples/tree/main/containers-orchestration/managed-kubernetes/autoscaler)
 
-$ kubectl get nodepools
-NAME            FLAVOR   AUTO SCALED   [...]   ANTI AFFINITY   DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   MIN   MAX
-nodepool-b2-7   b2-7     true          [...]   false           1         1         1            1           0     3
-```
+When you create your cluster, you can bootstrap a default node pool in it, and you can add others in the Public Cloud section of the [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB) or directly [using the Kubernetes API](/pages/public_cloud/containers_orchestration/managed_kubernetes/managing-nodes).
 
-Now we can begin to deploy a test workload on the cluster.
 
 ## Deploying a test workload
 
-In order to test the autoscaler, we propose you to install a *Prime Numbers* Deployment that deploys several instances of *Prime Numbers* pods. The *Prime Numbers* pod simply calculates prime numbers in the most performance ineffective way: by dividing every positive integer number by all the lower positive integers. It's a CPU intensive operation, but it uses a minimal amount of memory.  
+Let's assume that you have created an MKS cluster with a node pool with its minimum set to 1 and its maximum set to 3.
 
-> [!primary]
->
-> The *Prime Numbers* pod is a small [NodeJS](https://nodejs.org/) programs. The main script, on `index.js`, is a [Express server](https://expressjs.com/) that launches a [worker thread](https://nodejs.org/api/worker_threads.html) were the prime number calculation is done outside the event loop.
+In order to test the autoscaler, we offer you to install a *Python heavy CPU load* Deployment that deploys several instances of *Python CPU load* pods. The *Python CPU load* pod simply heavy charge each allocated CPU of your pod. It's a CPU intensive operation, but it uses a minimal amount of memory.
```

Comment:
"The *Python CPU load* pod simply heavy charge each allocated CPU of your pod."
Isn't a verb missing here ? I don't understand that sentence.

Reply:
yes agree :), is it better: "The *Python CPU Load* pod aims to consume all the CPU allocated to it" ?

Reply:
```suggestion
In order to test the autoscaler, we offer you to install a *Python heavy CPU load* Deployment that deploys several instances of *Python CPU load* pods. The *Python CPU load* pod's goal is to consume all the CPU allocated to it. It's a CPU intensive operation but it uses a minimal amount of memory.
```